### DISCUSSION TITLE
Welcome to magvit2-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/magvit2-pytorch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0nutation** [comment]
I write a pytorch version of Lookup-Free Quantization in https://github.com/0nutation/Lookup-Free-Quantization.

**Comment by jpfeil** [comment]
I'm a computational biologist trying to learn more about image tokenizers and how they can be used to solve problems in the biomedical sciences.

Since this model is very large, I was wondering if anyone has tried running this using multi-gpu training. Would something like deepspeed work for this?
